internal/cluster/distribution/eks/eksworkflow/workflow_update_node_pool.go

```diff
@@ -159,6 +162,8 @@ func (w UpdateNodePoolWorkflow) Execute(ctx workflow.Context, input UpdateNodePo
 			NodeVolumeEncryptionKeyARN  string         `mapstructure:"NodeVolumeEncryptionKeyARN,omitempty"`  // Note: NodeVolumeEncryptionKeyARN is only available from template version 2.1.0.
 			NodeVolumeSize              int            `mapstructure:"NodeVolumeSize"`
 			TemplateVersion             semver.Version `mapstructure:"TemplateVersion,omitempty"` // Note: TemplateVersion is only available from template version 2.0.0.
+			UseInstanceStore            string         `mapstructure:"UseInstanceStore"`          // Note: TemplateVersion is only available from template version 2.2.0.
```

Comment:
**Required**: `omitempty` to allow older stack parsing. (or make sure to test it works without it, I'm not 100% sure whether the refactored logic breaks without it or not. Gonna check it when I have the time.)

Reply:
Added omitempty.